lib/cli-engine/cli-engine.js

```diff
@@ -1037,11 +1037,7 @@ class CLIEngine {
             try {
                 return require(formatterPath);
             } catch (ex) {
-                if (format === "table" || format === "codeframe") {
-                    ex.message = `The ${format} formatter is no longer part of core ESLint. Install it manually with \`npm install -D eslint-formatter-${format}\``;
-                } else {
-                    ex.message = `There was a problem loading formatter: ${formatterPath}\nError: ${ex.message}`;
-                }
+                ex.message = format === "table" || format === "codeframe" ? `The ${format} formatter is no longer part of core ESLint. Install it manually with \`npm install -D eslint-formatter-${format}\`` : `There was a problem loading formatter: ${formatterPath}\nError: ${ex.message}`;
```

Comment:
Too bad we don't have prettier which would automatically split this onto multiple lines.

Reply:
I have found prettier to sometimes be frustrating to work with, but it does take care of things like this and I know is well-liked. Pending addition of any such tools, I've at least made line breaks for the ternary alternates.

Reply:
I would love to add Prettier (in a separate PR), but I won't be surprised if there's a reason why ESLint hasn't adopted it yet.